setup.py

```diff
@@ -41,6 +41,7 @@ def run_tests(self):
     },
     tests_require=['tox',
                    'pytest',
-                   'pytest-cov'],
+                   'pytest-cov',
```

Comment:
Are `pytest` and `pytest-cov` necessary here? If the tests are run through `tox`, then tox will create a virtualenv and run the tests in that, so it's the tox virtualenv that needs pytest and pytest-cov. You can do that with something like this in tox.ini:

```
deps =
    pytest
    pytest-cov
```